CMakeLists.txt

```diff
@@ -185,11 +195,12 @@ else()
 endif()
 
 include_directories(
-	src/common/
-	src/mac/
+  src/common/
+  src/mac/
   src/agora/
   src/agora/txrx/
   src/client/
+  /usr/include/mkl/
```

Comment:
This path is for the Ubuntu 20.04 mkl package.  I was anticipating this to not be an issue for 18.04 or a system without the distro package (installed through Parallel Studio).  I can remove if there is an issue, otherwise it might be useful for new users.